amethyst_renderer/Cargo.toml

```diff
@@ -39,6 +40,8 @@ rayon = "0.8"
 rayon-core = "1.2"
 serde = "1.0"
 serde_derive = "1.0"
+shred = "0.5"
+shrev = "0.6"
```

Comment:
Intended duping?

Reply:
Not a dupe!  Those are different crates.

Reply:
I'll stop reviewing, I'm starting to see different things as the same now. :(

Reply:
Nah, don't worry, to `Err` is human :P